components/feature/session/src/main/java/mozilla/components/feature/session/TrackingProtectionUseCases.kt

```diff
@@ -91,30 +91,34 @@ class TrackingProtectionUseCases(
          * (https://github.com/mozilla-mobile/android-components/issues/6283).
          */
         operator fun invoke() {
-            engine.trackingProtectionExceptionStore.removeAll(sessionManager.all.map {
```

Comment:
@Amejia481 In the comment above you mention #6283, which this patch fixes technically. However you mention a performance hit that we can avoid with browser store... what else would be needed here?

Reply:
The implementation is sufficient, I think we can remove the comment :)